api_app/analyzers_manager/observable_analyzers/onionscan.py

```diff
@@ -0,0 +1,35 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, ObservableAnalyzer
+
+
+class Onionscan(ObservableAnalyzer, DockerBasedAnalyzer):
+    name: str = "Onionscan"
+    url: str = "http://tor_analyzers:4004/onionscan"
```

Comment:
4004 is already used by APK_analyzers. Not a technical problem because it works but for clarity I would prefer this to have a different port from the other docker integrations. `4001` is free.

Reply:
Sure Sir!